config/initializers/rack_attack.rb

```diff
@@ -46,7 +46,7 @@ def path_without_extentions
   #
   # Key: "rack::attack:#{Time.now.to_i/:period}:req/ip:#{req.ip}"
 
-  throttle('req/ip', limit: ENV.fetch('RACK_ATTACK_LIMIT', 3000), period: 1.minute, &:ip)
```

Comment:
Any particular reason to change the default value? This would make this a breaking change

Reply:
Actually this has been changed earlier (#7866) with no explanation, so I thought it was set to 3000 by mistake as the number is so high for rate limiter.

Reply:
Thanks for pointing that out, I could have written a better PR description with more context. Anyway, we had a couple of complaints from users where their team would access the app from a single IP, perhaps behind a NAT layer, which caused a lot of `429` HTTP errors. We figured it made sense to increase this by default since it seems like a sane number for medium to large installations. Let's revert this back to 3000?